WireUI/Package.swift

```diff
@@ -10,35 +10,46 @@ let package = Package(
     defaultLocalization: "en",
     platforms: [.iOS(.v15), .macOS(.v12)],
     products: [
-        .library(name: "WireAccountImage", targets: ["WireAccountImage"]),
+        .library(name: "WireAccountImageUI", targets: ["WireAccountImageUI"]),
+        .library(name: "WireConversationListUI", targets: ["WireConversationListUI"]),
         .library(name: "WireDesign", targets: ["WireDesign"]),
+        .library(name: "WireMainNavigationUI", targets: ["WireMainNavigationUI"]),
         .library(name: "WireReusableUIComponents", targets: ["WireReusableUIComponents"]),
-        .library(name: "WireSidebar", targets: ["WireSidebar"]),
-        .library(name: "WireMainNavigation", targets: ["WireMainNavigation"])
+        .library(name: "WireSettingsUI", targets: ["WireSettingsUI"]),
+        .library(name: "WireSidebarUI", targets: ["WireSidebarUI"])
     ],
     dependencies: [
         .package(url: "https://github.com/apple/swift-docc-plugin", from: "1.1.0"),
         .package(name: "WireFoundation", path: "../WireFoundation")
     ],
     targets: [
+        .target(name: "WireAccountImageUI", dependencies: ["WireFoundation"]),
+        .testTarget(name: "WireAccountImageUITests", dependencies: ["WireAccountImageUI", "WireFoundation"]),
+
+        .target(name: "WireConversationListUI"),
+        .testTarget(name: "WireConversationListUITests", dependencies: ["WireConversationListUI"]),
+
         .target(name: "WireDesign", dependencies: ["WireFoundation"]),
-        .testTarget(name: "WireDesignTests", dependencies: ["WireDesign", WireTestingPackage]),
+        .testTarget(name: "WireDesignTests", dependencies: ["WireDesign"]),
 
-        .target(name: "WireReusableUIComponents", dependencies: ["WireDesign", "WireFoundation"]),
-        .testTarget(name: "WireReusableUIComponentsTests", dependencies: ["WireReusableUIComponents", WireTestingPackage]),
+        .target(name: "WireMainNavigationUI"),
+        .testTarget(name: "WireMainNavigationUITests", dependencies: ["WireMainNavigationUI"]),
 
-        .target(name: "WireMainNavigation", dependencies: ["WireDesign"]),
-        .testTarget(name: "WireMainNavigationTests", dependencies: ["WireMainNavigation", WireTestingPackage]),
+        .target(name: "WireReusableUIComponents", dependencies: ["WireDesign", "WireFoundation"]),
```

Comment:
🤔 I'm wondering if this package should have any dependencies to WireFoundation. Maybe it'd make sense to move any shared UI code from WireFoundation to this package. I see that WireFoundation has a Design folder, this could possibly be moved to this package.

Reply:
Yes, I hope we can find a nice solution within our architecture forum.
In this PR this I have only rearranged the targets alphabetically, but didn't change and the dependencies of WireReusableUIComponents.